cg/meta/clean/clean_flow_cells.py

```diff
@@ -173,9 +170,7 @@ def get_files_for_samples_on_flow_cell_with_tag(self, tag: str) -> list[File] |
                     bundle_name=bundle_name, tags=[tag, self.flow_cell.id]
                 ).all()
             )
-            if not files:
```

Comment:
Not very clear from here but one level of indentation was removed in this if statement